xwrf/config.yaml

```
@@ -22,9 +22,20 @@ time_coords:
   - Time
   - time
 
-boolean_units_attrs:
-  - '-'
-  - flag
+unit_harmonization_map:
+  kelvin:
+    - Kelvin
+  dimensionless:
```

```suggestion
  '1':
```

If we wanted to make both Pint and CF/UDUNITS happy, I think this needs to be "1" (http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#units)